src/probeinterface/wiring.py

```diff
@@ -61,7 +61,7 @@ def get_available_pathways():
     return list(pathways.keys())
 
 
-def wire_probe(probe, pathway, channel_offset=0):
+def wire_probe(probe: "Probe", pathway: str, channel_offset: int = 0):
```

Comment:
Does importing Probe leads to circular import thing? Have you tried the [from __future__ import annotations](https://stackoverflow.com/questions/61544854/from-future-import-annotations) trick?

I kind of just want to refresh this on my own mind.

Reply:
I think doing the string notation prevents this. So ideally I would give it an actually type but in this case it will just tell the user it is a "Probe" not a probeinterface.Probe, but just the word Probe. It is a way to get around something like a circular import. I do that with DataFrame later since we only import the `pandas` inside of the function and so it isn't in the "typing" namespace.

Reply:
Right. That's true. I remember I encountered this issue in spikeinterface and had to do some hack to fix it. In retrospective that was wrong and should have sticked with strings but I wanted to be able to click and navigate within my IDE:

https://github.com/h-mayorquin/spikeinterface/blob/5e420f3a847102c145c705dddfb01b140b318ec3/src/spikeinterface/extractors/nwbextractors.py#L11-L18